addons/website_payment/static/src/snippets/s_donation/options.js

```diff
@@ -11,7 +11,10 @@ options.registry.Donation = options.Class.extend({
      */
     start() {
         this.defaultDescription = _t("Add a description here");
-        return this._super(...arguments);
+        this.$target[0].querySelector(".s_donation_donate_btn").classList.add("oe_unremovable")
+        // Making 'not-editable' donation button to ensure consistent behavior with the master branch.
+        this.$target[0].querySelector(".s_donation_donate_btn").classList.add("o_not_editable")
```

Comment:
Can't we add both of this class ```oe_unremovable``` and ```o_not_editable``` in xml [here](https://github.com/odoo/odoo/blob/15.0/addons/website_payment/views/snippets/s_donation.xml#L24)